cf-testd/cf-testd.c

```diff
@@ -471,12 +542,49 @@ int main(int argc, char *argv[])
     if (config->file != NULL)
     {
         Log(LOG_LEVEL_NOTICE, "Got file argument: '%s'", config->file);
-        Log(LOG_LEVEL_ERR, "File input not yet supported");
-        // TODO: implement file input
-        exit(EXIT_FAILURE);
+        FILE *fp = NULL;
```

Comment:
fopen here instead.

Reply:
Then it can also be const:
```C
FILE *const fp = fopen(config->file, "r");
```

Reply:
(Make sure this line is directly above the if test, so `GetErrorStr()` is correct)